documentation/modules/auxiliary/scanner/snmp/snmp_enumshares.md

```diff
@@ -0,0 +1,32 @@
+## Description
+This module will simply scan a range of hosts and queries via SNMP to determine any available shares.
+
```

Comment:
Generally I like to document how you created the vulnerability here so that others (maybe making a lab, or verifying this module works) can replicate.  Most likely: https://support.microsoft.com/en-us/help/324263/how-to-configure-the-simple-network-management-protocol-snmp-service-i